connectors/sqs/src/main/java/io/camunda/connector/inbound/SqsQueueConsumer.java

```diff
@@ -38,28 +39,36 @@ public SqsQueueConsumer(
   @Override
   public void run() {
     LOGGER.info("Started SQS consumer for queue {}", properties.getQueue().getUrl());
+
     final ReceiveMessageRequest receiveMessageRequest = createReceiveMessageRequest();
     ReceiveMessageResult receiveMessageResult;
     do {
       receiveMessageResult = sqsClient.receiveMessage(receiveMessageRequest);
       List<Message> messages = receiveMessageResult.getMessages();
       for (Message message : messages) {
-        InboundConnectorResult<?> correlate =
-            context.correlate(MessageMapper.toSqsInboundMessage(message));
-        if (correlate.isActivated()) {
+        try {
+          correlate(message);
           sqsClient.deleteMessage(properties.getQueue().getUrl(), message.getReceiptHandle());
-          LOGGER.debug(
-              "Inbound event correlated successfully: {}, and message with ID {} was deleted",
-              correlate.getResponseData(),
-              message.getMessageId());
-        } else {
-          LOGGER.debug("Inbound event not correlated: {}", correlate.getErrorData());
+        } catch (ConnectorInputException e) {
+          LOGGER.warn("NACK (no requeue) - failed to parse SQS message body: {}", e.getMessage());
+        } catch (Exception e) {
+          LOGGER.debug("NACK (requeue) - failed to correlate event", e);
```

Comment:
One exception statement says `no requeue`, the other one `requeue` but they both do nothing. Is that expected?